doc/lvm.md

```diff
@@ -0,0 +1,168 @@
+# LVM features and YaST
+
+The basic LVM functionality has been supported by YaST for ages. But LVM has a lot of features
+like RAID, snapshots, thin provisioning and whatsnot that often result in special types of logical
+volumes being present in the system.
+
+This document summarizes how well can YaST deal with systems that contain such advanced LVM setups.
+
+## The Partitioner
+
+This section describes how the Expert Partitioner represent the different LVM technologies and what
+operations it allows for each type of logical volume. At the current stage, some operations show an
+unexpected behavior and, in most cases, would need to be adjusted. That is represented in bold text.
+
+Note that, unlike RAID0, stripped LVs are not really a separate type. Many types of LVs can be
+stripped.
+
+### Normal LVM
+
+This is somehow the base case. Normal LVs are visualized in the standard way and all operations
+are supported.
```

Comment:
Resizing it not allowed if the logical volume has snapshots and is active. Thus we decided to not allow it.

Reply:
Added a note about that.